python/ray/data/_internal/execution/operators/one_to_one_operator.py

```diff
@@ -10,7 +10,31 @@
 from ray.types import ObjectRef
 
 
-class LimitOperator(PhysicalOperator):
+class OneToOneOperator(PhysicalOperator):
+    """A streaming operator that executes once its input is ready.
```

Comment:
This comment doesn't look right. OneToOneOperator should mean that it has one input and one output.